tests/e2e/standard/contacts/create-places-contacts.wdio-spec.js

```diff
@@ -0,0 +1,44 @@
+const loginPage = require('../../../page-objects/default/login/login.wdio.page');
+const commonPage = require('../../../page-objects/default/common/common.wdio.page');
+const contactPage = require('../../../page-objects/standard/contacts/contacts.wdio.page');
+
+const PLACE_TYPES = { CLINIC: 'clinic', HEALTH_CENTER: 'health_center', DISTRICT: 'district_hospital' };
+const CONTACT_NAME_SUFFIX = ' Contact';
+
+describe('Creating contacts with standard config. ', () => {
+  const districtName = 'District test';
```

Comment:
You can add all constants here since they won't be used outside this scope.
Then on top will be just the imports